Examples/Cocoa/Sources/Objective_C/Board.m

```diff
@@ -291,6 +308,16 @@ - (NSDictionary *)dictionaryRepresentation
             [dict setObject:[NSNull null] forKey:@"created_at"];
         }
     }
+    if (_boardDirtyProperties.BoardDirtyPropertyContributors) {
+        NSSet *items0 = _contributors;
+        NSMutableSet *result0 = [NSMutableSet setWithCapacity:items0.count];
+        for (id obj0 in items0) {
+            if (obj0 != (id)kCFNull) {
+                [result0 addObject:[obj0 dictionaryRepresentation]];
```

Comment:
I think we're going to have to add a cast here to avoid hitting warnings for strict selector matching. Added an issue here https://github.com/pinterest/plank/issues/90